contributing/documentation/overview.rst

```diff
@@ -233,62 +204,34 @@ steps to contribute to the Symfony documentation, which you can use as a
     $ git push origin my_changes
 
     # go to GitHub and create the Pull Request
-    #
-    # Include this table in the description:
-    # | Q             | A
-    # | ------------- | ---
-    # | Doc fix?      | [yes|no]
-    # | New docs?     | [yes|no] (PR # on symfony/symfony if applicable)
-    # | Applies to    | [Symfony version numbers this applies to]
-    # | Fixed tickets | [comma separated list of tickets fixed by the PR]
 
     # (optional) make the changes requested by reviewers and commit them
     $ git commit xxx.rst
     $ git push
 
-You guessed right: after all this hard work, it's **time to celebrate again!**
-
+After completing your next contributions, you can also see how your ranking
+improves in the list of `Symfony Documentation Contributors`_. You guessed
+right: after all this hard work, it's **time to celebrate again!**
 
 Review your changes
 -------------------
 
 Every GitHub Pull Request is automatically built and deployed by `Platform.sh`_
-on a single environment that you can access on your browser to review your 
+on a single environment that you can access on your browser to review your
 changes.
 
 .. image:: /images/contributing/docs-pull-request-platformsh.png
    :align: center
    :alt:   Platform.sh Pull Request Deployment
 
-To access the `Platform.sh`_ environment URL, simply go to your Pull Request 
-page on GitHub and click on ``Details``.
-
-.. note::
-
-    The specific configuration files at the root of the Git repository: 
-    ``.platform.app.yaml``, ``.platform/services.yaml`` and 
-    ``.platform/routes.yaml`` allow `Platform.sh`_ to build Pull Requests.
+To access the `Platform.sh`_ environment URL, go to your Pull Request page on
+GitHub and click on ``Details``.
```

Comment:
Since some time you must first expand the list of tests on GitHub by clicking on the "Show all checks" text.